assets/home/notices.scss

```diff
@@ -28,6 +33,19 @@
 					white-space: nowrap;
 				}
 			}
+
+			.button-secondary {
+				color: #155E65;
```

Comment:
It seems a little unexpected to me that the text in these buttons is different from the border. Was this a deliberate design choice?

---
<img width="278" alt="Screenshot 2022-12-12 at 16 25 59" src="https://user-images.githubusercontent.com/842193/207147321-017631bf-de9b-416c-94cb-27be73ee248a.png">